experiments/conductor/branches.yaml

```diff
@@ -13,49 +13,31 @@
 # limitations under the License.
 
 branches:
-  - name: accessapproval-approvalrequest
```

Comment:
Probably should try avoiding stomping on each others branches.yaml. Maybe add it to the .gitignore?

Reply:
I added it to .gitignore, but it still shows up as diff (may be because the file exists in the repo).